test/dummy_backend.cpp

```diff
@@ -0,0 +1,82 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "ngraph/descriptor/layout/dense_tensor_layout.hpp"
+#include "ngraph/except.hpp"
+#include "ngraph/op/convert.hpp"
+#include "ngraph/op/select.hpp"
+#include "ngraph/op/util/binary_elementwise_comparison.hpp"
+#include "ngraph/pass/assign_layout.hpp"
+#include "ngraph/pass/like_replacement.hpp"
+#include "ngraph/pass/liveness.hpp"
+#include "ngraph/pass/manager.hpp"
+#include "ngraph/runtime/backend_manager.hpp"
+#include "ngraph/util.hpp"
+
+#include "test/dummy_backend.h"
+
+using namespace std;
+namespace ng = ngraph;
+
+namespace ngraph {
+
+using descriptor::layout::DenseTensorLayout;
+runtime::dummy::DummyBackend::DummyBackend() {
+  cout << "Dummy Backend constructor called " << endl;
```

Comment:
stray `cout`

Reply:
It's not stray, I left it there on purpose.